tapir/statistics/views.py

```diff
@@ -206,6 +217,62 @@ def get_context_data(self, **kwargs):
         return context_data
 
 
+class AverageBasketEvolutionJsonView(LoginRequiredMixin, JSONView):
+    def get_context_data(self, **kwargs):
+        tapir_user = get_object_or_404(TapirUser, pk=(self.kwargs["pk"]))
+        logged_user = self.request.user
+        if tapir_user.pk != logged_user.pk and not logged_user.has_perm(
+            PERMISSION_COOP_MANAGE
+        ):
+            return HttpResponseForbidden("Access forbidden")
+
+        user_purchases = (
+            PurchaseBasket.objects.filter(tapir_user=tapir_user)
+            .annotate(month=TruncMonth("purchase_date"))
+            .values("month")
+            .annotate(average_gross_amount=Avg("gross_amount"))
+            .order_by("month")
+        )
+
+        months = self.get_months(user_purchases)
+
+        return build_line_chart_data(
+            x_axis_values=months,
+            y_axis_values=[self.get_avg_price_per_month(user_purchases, months)],
+            data_labels=[_("Price of average basket")],
+        )
+
+    @staticmethod
+    def get_months(user_purchases):
+        months = []
+        if len(user_purchases) > 0:
+            now = datetime.datetime.now().replace(
```

Comment:
I would start from `timezone.now()`.
Also, I think you can shorten the conversion a bit by converting to a `date` instead of a `datetime`.
So this line could be :
```python
now = timezone.now().date().replace(day=1)
```